natlas-server/entrypoint.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/sh
 
-mkdir -p /data/{db,media}
+mkdir -p /data/{db,media,logs}
+ln -s /data/logs logs
```

Comment:
@0xdade/anybody, is it possible to tell flask to write to stdout, not a log file? Following the twelve factor app model (https://12factor.net/logs), logs should go to stdout or some sort of structured log endpoint.

Reply:
Technically we could do this but right now we're not using an app logger and the only logs from flask are in f.write() statements. The agent sends logs to both stdout and a rotating file handler. There's an open issue to move to proper logging, the server is the only thing that needs it now.

Reply:
Alright how about we table this conversation for that other issue. I do want to get this PR merged.

#67

Reply:
@ajacques This is something that we need bad. Just like tests. Apparently flask has a default builtin logger that can be invoked via app.logger.info("log this"), but I dont know how and if we can interface with it.

Reply:
Also, a lot of containers will just symlink their logfile to the /dev/stdout. Then disable log rotation.